rock.osdeps

```diff
@@ -411,3 +411,12 @@ gawk:
 
 bzip2:
     debian,ubuntu: libbz2-dev
+
+slam/pcl:
+    ubuntu:
+        default: libpcl-dev
+        '14.04': libpcl-1.7-all-dev
```

Comment:
this is the COMPLETE /etc/apt/sources.list of 14.04

(14.04_amd64)root@buildsrv01:/home/planthaber# cat /etc/apt/sources.list
deb http://archive.ubuntu.com/ubuntu trusty main universe multiverse
#deb http://archive.ubuntu.com/ubuntu precise main universe multiverse